lib/matplotlib/markers.py

```diff
@@ -201,7 +201,7 @@ class MarkerStyle:
     # TODO: Is this ever used as a non-constant?
     _point_size_reduction = 0.5
 
-    def __init__(self, marker=None, fillstyle=None):
+    def __init__(self, marker=None, fillstyle=None, *, normalize_path=True):
```

Comment:
I think just `normalize` is enough?

Reply:
I named it `normalize_path` to give an indication that this (currently?) affects only custom paths. Of course it's up for discussion; maybe the same argument could be used in the future to also normalize inbuilt markers?